src/PythonBindings/VectorPyList.hpp

```diff
@@ -0,0 +1,39 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <vector>
+
+#include <boost/python.hpp>
+#include <boost/python/stl_iterator.hpp>
+#include <boost/python/suite/indexing/vector_indexing_suite.hpp>
+
+namespace bp = boost::python;
+/*!
+ * \ingroup PythonBindingsGroup
+ * \brief Convert a bp::list of bp::object& to a std::vector
```

Comment:
We've been preferring ticks around types, e.g. `bp::list` and `bp::object&` as well as `std::vector`